.github/workflows/deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Add AWS CLI
+      run: apt-get update && apt-get install -y awscli
+
+    - name: Configure AWS CLI
+      run: |
+        aws --profile poser configure set region $AWS_REGION
```

Comment:
I'm wondering why this is needed, as the SDK should be able to get the values from the ENV variables directly ..